content/docs/user-guide/external-dependencies.md

```diff
@@ -41,64 +41,57 @@ stage to your list of stages in dvc.yaml.
 > Note that some of these commands use the `/home/shared` directory, typical in
 > Linux distributions.
 
-### Local file system path
-
-```dvc
-$ dvc run -n download_file
-          -d /home/shared/data.txt \
-          -o data.txt \
-          cp /home/shared/data.txt data.txt
-```
-
-### SSH
-
-```dvc
-$ dvc run -n download_file
-          -d ssh://user@example.com:/home/shared/data.txt \
-          -o data.txt \
-          scp user@example.com:/home/shared/data.txt data.txt
-```
-
 ### Amazon S3
 
 ```dvc
 $ dvc run -n download_file
-          -d s3://mybucket/data.txt \
-          -o data.txt \
-          aws s3 cp s3://mybucket/data.txt data.txt
+          -d s3://mybucket/data \
+          -o data \
+          aws s3 cp s3://mybucket/data data
 ```
 
 ### Microsoft Azure Blob Storage
 
 ```dvc
 $ dvc run -n download_file
-          -d azure://my-container-name/data.txt \
-          -o data.txt \
+          -d azure://my-container-name/data \
+          -o data \
           az storage copy \
                      -d data.json \
                      --source-account-name my-account \
                      --source-container my-container-name \
-                     --source-blob data.txt
+                     --source-blob data
 ```
 
 ### Google Cloud Storage
 
 ```dvc
 $ dvc run -n download_file
-          -d gs://mybucket/data.txt \
-          -o data.txt \
-          gsutil cp gs://mybucket/data.txt data.txt
+          -d gs://mybucket/data \
+          -o data \
+          gsutil cp gs://mybucket/data data
 ```
 
+### SSH
+
+```dvc
+$ dvc run -n download_file
+          -d ssh://user@example.com/path/from/sftp/root/to/data \
+          -o data \
+          scp user@example.com:/path/from/sftp/root/to/data data
```

Comment:
not sure that scp is using sftp and it's the same convention ... how often sftp root is not /?

Reply:
Good point about `scp` not being SFTP! Missed that 🤦 I'm going to remove these stfp mentions from the paths... ⌛ 

>how often sftp root is not /?

Not sure but it's a note we already have in remote add so it has been deemed important before. Plus per this https://github.com/iterative/dvc/issues/4167#issuecomment-663318837 I thought we definitely wanted to mention this clearly in all the docs where it may cause issues.

Reply:
OK so... I'm removing all the `/from/sftp/...` paths again. Having the note underneath should be enough I think.

In fact, should we just remove these SFTP notes completely? And or consider renaming `ssh://` to `sftp://` if that's the actual protocol we're using... Then users could be expected to know what kinds of URLs and paths are supported. Cc @efiop

Reply:
> In fact, should we just remove these SFTP notes completely? 

Could just create 1 note in `remote modify/add` help. The rest could be removed, yes.

> nd or consider renaming ssh:// to sftp:// if that's the actual protocol we're using...

Not really worth renaming. Plus, we do use ssh too, but only for external data management, so it is not a direct swap. Would just keep as is for now.

Reply:
I left the notes everywhere just in case. It's a shorter note version now, anyway.

Agreed about not renaming. Thanks @efiop